### PR TITLE
disable converse completely if using gui mode as its totally broken

### PR DESCRIPTION
**Pull request by AIIX**
- Disabled converse method completely if using the GUI fixes https://github.com/OpenVoiceOS/skill-ovos-setup/issues/48

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -502,12 +502,13 @@ def handle_mycroft_ready(self, message):
 
     # voice events
     def converse(self, message):
-        if self.state != SetupState.INACTIVE or \
-                self.state != SetupState.FIRST_BOOT:
-            # capture all utterances until paired
-            # prompts from this skill are handled with get_response
-            return True
-        return False
+        if not is_gui_running() or not can_use_touch_mouse():
```
Comment (JarbasAl) [inline __init__.py:505]: should instead check if pairing mode is GUI or not, regardless of mouse etc we dont want this in GUI pairing mode

Comment (AIIX) [reply __init__.py]: uses pairing mode check now

File: __init__.py [hunk 2]
```diff
@@ -502,12 +502,13 @@ def handle_mycroft_ready(self, message):
...
+            if self.state != SetupState.INACTIVE or \
+                    self.state != SetupState.FIRST_BOOT:
```
Comment (JarbasAl) [inline __init__.py:507]: should be `if self.state != SetupState.INACTIVE and self.state != SetupState.FINISHED`

Comment (AIIX) [reply __init__.py]: changed it

File: __init__.py [hunk 3]
```diff
@@ -502,12 +502,13 @@ def handle_mycroft_ready(self, message):
...
     def converse(self, message):
-        if self.state != SetupState.INACTIVE or \
-                self.state != SetupState.FIRST_BOOT:
-            # capture all utterances until paired
-            # prompts from this skill are handled with get_response
-            return True
-        return False
+        if self.pairing_mode != PairingMode.GUI:
+            if self.state != SetupState.INACTIVE and \
+                self.state != SetupState.FINISHED
```
Comment (JarbasAl) [inline __init__.py:507]: see failing unittests, missing a `:`

Comment (AIIX) [reply __init__.py]: should be fixed

**Pull request merged**